src/webview/mod.rs

```diff
@@ -232,6 +232,8 @@ pub struct WebViewAttributes {
 
   /// Whether all media can be played without user interaction.
   pub autoplay: bool,
+
+  pub on_load_handler: Option<Box<dyn Fn(&Window, Url)>>,
```

Comment:
Avoid using `Url` for consistency with other handlers
```suggestion
  pub on_page_load_handler: Option<Box<dyn Fn(&Window, String)>>,
```

Reply:
I'm not sure how this would make it more consistent. The APIs that take a url as a parameter do so as a `&str`, but everywhere that a url is returned or stored it is done so as a `url::Url`, given that this is data been returned Wry to the client of the API is seems like `url::Url` would be the most consistent choice. None of the public interactions with urls use `String` that I can find.

Reply:
all of these handlers take the url as a String https://github.com/tauri-apps/wry/blob/4ab2a45a6a7cc1233c00c008c8808606edd4cbf4/src/webview/mod.rs#L159-L188

Reply:
I guess all callbacks do. I'll make that change.

Reply:
Should I take out the &Window parameter also? None of the handlers you pointed out take the Window as a parameter and I ended up not being able to use it from the tauri side like I had anticipated.

Reply:
Sure, we can add it later if needed